lib/services/amazonka-dynamodb-streams/amazonka-dynamodb-streams.cabal

```diff
@@ -87,11 +87,7 @@ test-suite amazonka-dynamodb-streams-test
   -- This section is encoded by the template and any modules added by
   -- hand outside these namespaces will not correctly be added to the
   -- distribution package.
-  other-modules:
-    Test.Amazonka.DynamoDBStreams
-    Test.Amazonka.DynamoDBStreams.Internal
-    Test.Amazonka.Gen.DynamoDBStreams
-
+  other-modules:    Test.Amazonka.Gen.DynamoDBStreams
```

Comment:
Did you have to edit the cabal files by hand? It's probably worth regenerating them with `scripts/generate dynamodb dynamodbstreams` once you make the necessary config edits.

Reply:
Ah yeah, mostly; though I also staged the changes from the commit hook later. I'll run the script for all three.